### PR TITLE
Add an alt attribute to all extension thumbnails | Fixes #138

### LINKED ISSUE #138: Most images are missing an alt attribute
When looking through the code for the homepage and extensions page, I noticed that most of the extension images don't have an alt attribute.

### PR DESCRIPTION
**Pull request by kerrtravers**
Fixes #138

#### Changes
- Adds an alt attribute to all the extension thumbnails.
- Changes a scratch link to https and fixes two stray ``&``'s.

#### Tests 
I set up an environment to make sure I haven't broken anything, Everything works as expected.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -257,7 +257,7 @@ <h2>Fischertechnik robots</h2>
                                 <h2>Intel&reg; RealSense&trade;</h2>
                                 <p class="author">Shachar Oz, Yaron Yanai, Yotam Salmon</p>
                             </header>
-                            <img src="images/extensions/realsense_small.png" />
+                            <img src="images/extensions/realsense_small.png" alt="ischertechnik robots" />
```
Comment (thisandagain) [inline index.html]: This should be "Intel RealSense"

**Review by thisandagain — CHANGES_REQUESTED** [review]

**Comment by thisandagain** [comment]
Looks good @vutondesign. Just one small issue.

File: index.html (hunk 1 above)
Comment (kerrtravers) [reply index.html]: Whoops

**Comment by kerrtravers** [comment]
Sorry for the late reply, that should be fixed now. 👍

**Review by thisandagain — APPROVED** [review]

**Comment by thisandagain** [comment]
Thanks @vutondesign !

**Pull request merged**